.github/workflows/test-core.yaml

```diff
@@ -40,13 +53,14 @@ jobs:
       - uses: magnetikonline/action-golang-cache@v1
         with:
           go-version: ${{env.GO_VERSION}}
```

Comment:
Not part of this work, so feel free to ignore it, but we could read the Go version from the `.go-version` file like the release workflow.

Reply:
yeah we should be able to set `go-version-file: go.mod` instead; i'm gonna try that in a follow up pr